deploy/kubernetes/elastic-agent-kustomize/default/elastic-agent-standalone/base/elastic-agent-standalone-daemonset-configmap.yaml

```diff
@@ -28,8 +28,9 @@ data:
     providers.kubernetes:
       node: ${NODE_NAME}
       scope: node
-      #Uncomment to enable hints' support
+      #Uncomment to enable hints' support - https://www.elastic.co/guide/en/fleet/current/hints-annotations-autodiscovery.html
       #hints.enabled: true
+      #hints.default_container_logs: true
```

Comment:
is it required to set it explicitly? default value is `true` https://github.com/elastic/elastic-agent/blob/v8.14.3/internal/pkg/composable/providers/kubernetes/config.go#L69

Reply:
Indeed the default is true. Not needed to be set.
It is something like the kubernetes provider that is enabled by default.

I want to avoid putting `hints.default_container_logs: false`, users accidentally uncomment and break default fucntionality